integration-tests/validator/src/main/resources/application.properties

```diff
@@ -15,4 +15,5 @@
 ## limitations under the License.
 ## ---------------------------------------------------------------------------
 
-quarkus.native.additional-build-args =-H:ResourceConfigurationFiles=resources-config.json
\ No newline at end of file
+quarkus.native.additional-build-args =-H:ResourceConfigurationFiles=resources-config.json
+quarkus.ssl.native=true
```

Comment:
I see that this is needed for accessing the GitHub https URL. But I wonder how should we help end users to get around this. Maybe we should just document it in `usage.adoc`, something like 

```
If you are accessing XSD resources via HTTPS (e.g. `from(...).to("validator:https://acme.org/foo/schema.xsd")`) than you need to add `quarkus.ssl.native = true` to your `application.properties`.
```

Hardcoding SSL support (like we do in some other extensions) seems to be an overkill in this case, IMO. I guess users will mostly use classpath resources. WDYT @jamesnetherton?